sdk/samples/iot/aziot_esp32/AzIoTSasToken.cpp

```diff
@@ -0,0 +1,242 @@
+#include "AzIoTSasToken.h"
+#include "SerialLogger.h"
```

Comment:
Curious why C++ for this sample instead of C.

Reply:
he is using C++ namespaces and classes

Reply:
Yes I see he is, but why choose this approach?  It is the only C++ in the entire repository.

Reply:
Since these are Arduino samples, I went with a c++ approach to make it more integrated.